torch/csrc/jit/runtime/static/accelerant.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <ATen/core/interned_strings.h>
+#include <ATen/core/ivalue.h>
+#include <torch/csrc/jit/api/module.h>
+#include <torch/csrc/jit/ir/ir.h>
+#include <torch/csrc/jit/passes/constant_propagation.h>
+#include <torch/csrc/jit/passes/inliner.h>
+
+class Accelerant {
```

Comment:
Can we just call this StatisRuntime or something. I can imagine it will draw confusion with accelerator. :)

Reply:
good point!

Reply:
done -- my attempt at word association (torch -> fire -> accelerant) landed me too close to an already used latin root 😛

Reply:
catalyst :)